src/partners/changehero.ts

```diff
@@ -1,4 +1,3 @@
-import axios from 'axios'
```

Comment:
Is axios used anywhere else? Can we delete it from the repo then? It looks Simplex is the only hit when I do a code search.

Reply:
yep just Simplex but they still need to be ported to fetch